src/components/tutorial-card/helpers/build-aria-label.ts

```diff
@@ -0,0 +1,42 @@
+import { ProductOption } from 'lib/learn-client/types'
+import { TutorialCardProps } from '../types'
+
+const PRODUCT_LABEL_MAP: Record<ProductOption, string> = {
```

Comment:
I've duplicated this for now (there's an identical map in `CardBadges`).

I think I might have a relatively high tolerance for duplication, but happy to try to abstract this within this PR, or in a follow-up, if we want. I think `productSlugToName` will be something we do a _lot_, but also feels like a can of worms to try to abstract it fully right at this moment given the variety of similar types.